src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoPercentNegativePattern.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class NumberFormatInfoPercentNegativePattern
+    {
+        // PosTest1: Verify default value of property PercentNegativePattern
+        [Fact]
+        public void TestDefault()
+        {
+            NumberFormatInfo nfi = new NumberFormatInfo();
+            int expected = nfi.PercentNegativePattern;
+            Assert.Equal(0, expected);
+        }
+
+        // PosTest2: Verify set value of property PercentNegativePattern
+        [Fact]
+        public void TestSetValue()
+        {
+            NumberFormatInfo nfi = new NumberFormatInfo();
+            for (int i = 0; i <= 11; i++)
+            {
+                nfi.PercentNegativePattern = i;
+                Assert.Equal(i, nfi.PercentNegativePattern);
+            }
+        }
+
+        // TestArgumentOutOfRange: ArgumentOutOfRangeException is thrown
+        [Fact]
+        public void TestArgumentOutOfRange()
+        {
+            VerificationHelper<ArgumentOutOfRangeException>(-1);
+            VerificationHelper<ArgumentOutOfRangeException>(12);
+        }
+
+        // TestInvalidOperation: InvalidOperationException is thrown
+        [Fact]
+        public void TestInvalidOperation()
+        {
+            NumberFormatInfo nfi = new NumberFormatInfo();
+            NumberFormatInfo nfiReadOnly = NumberFormatInfo.ReadOnly(nfi);
+            Assert.Throws<InvalidOperationException>(() =>
+            {
+                nfiReadOnly.PercentNegativePattern = 1;
+            });
+        }
+
+        // TestPercentNegativePatternLocale: Verify value of property PercentNegativePattern for specific locales
+        [Theory]
+        [InlineData("en-US", 0, 1)]
+        public void TestPercentNegativePatternLocale(string locale, int expectedWindows, int expectedIcu)
+        {
+            CultureInfo myTestCulture = new CultureInfo(locale);
+            NumberFormatInfo nfi = myTestCulture.NumberFormat;
+            int actual = nfi.PercentNegativePattern;
+            int expected = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? expectedWindows : expectedIcu;
+            // todo: determine why some values are different
```

Comment:
I will make a pass after this is merged. I assume this is because windows and ICU pull from different data sources (ICU uses the CLDR, Windows has their own data but are slowly moving towards ICU)